pkg/flare/strip.go

```diff
@@ -27,23 +28,25 @@ var replacers []replacer
 
 func init() {
 	apiKeyReplacer = replacer{
-		regex: regexp.MustCompile(`(?m)^\s*api_key:( *\w+(\w{5}) ?,?)+$`),
+		regex: regexp.MustCompile(`(?m)^\s*api_key:( *"?\w+(\w{5})"? ?,?)+$`),
```

Comment:
yaml strings can also be quoted with single quotes (`'`). We should handle this case too.

I don't mind dropping the "smart" regex that tries to keep the last 5 characters. Or if it makes things simpler, we could keep the last 5 characters without caring about what types of characters they are.

Reply:
You're right, I missed that. I deleted the 5 chars matched group since we don't even use that to get the tailing part of the key.